app/views/reminder_mailer/email_verification.text.erb

```diff
@@ -6,4 +6,4 @@ This is your 6-digit one time password:
 
 It is valid for <%= one_time_password_validity_duration %>. We recommend you copy and paste this into the text field on the email verification screen in the service.
 
-If you have any issues with the password, email us at: <%= support_email_address("early-career-payments") %>.
+If you have any issues with the password, email us at: <%= support_email_address() %>.
```

Comment:
Not sure here, but is this even in use?

Reply:
Yes it is in use: https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/app/controllers/claims_controller.rb#L237 and https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/app/controllers/reminders_controller.rb#L137

Reply:
I've opened a separate PR to remove these redundant templates: https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2628